OrbitLinuxTracing/include/OrbitLinuxTracing/Events.h

```diff
@@ -80,25 +80,27 @@ class Callstack {
 class FunctionCall {
  public:
   FunctionCall(pid_t tid, uint64_t virtual_address, uint64_t begin_timestamp_ns,
-               uint64_t end_timestamp_ns, uint32_t depth)
+               uint64_t end_timestamp_ns, uint32_t depth, uint64_t return_value)
       : tid_(tid),
         virtual_address_(virtual_address),
         begin_timestamp_ns_(begin_timestamp_ns),
         end_timestamp_ns_(end_timestamp_ns),
-        depth_{depth} {}
+        depth_{depth}, return_value_(return_value) {}
 
   pid_t GetTid() const { return tid_; }
   uint64_t GetVirtualAddress() const { return virtual_address_; }
   uint64_t GetBeginTimestampNs() const { return begin_timestamp_ns_; }
   uint64_t GetEndTimestampNs() const { return end_timestamp_ns_; }
   uint32_t GetDepth() const { return depth_; }
+  uint64_t GetReturnValue() const { return return_value_; }
```

Comment:
I would add a comment explaining how this is the return value only in some cases (such as when the return value fits in a register) and (approximately) which cases.

Reply:
Changed name to GetIntegerReturnValue.  I think even if the return value does not fit in register, RAX will hold the address of the returned object.